src/layouts/Layout.astro

```diff
@@ -101,7 +101,7 @@ const {
   </head>
   <body>
     <Header />
-    <main class="grow py-14">
+    <main class="py-14">
```

Comment:
`grow` appears to be a flex class, but without a flex container, it doesn't make sense. So, I've removed it. Unrelated to the issue though.